### DISCUSSION TITLE
checkbox in grid

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#5855] [Help]
hi This is my code but it does not work properly
and just one row checked checkbox.
`  public rowSelection: Serenity.GridRowSelectionMixin;

        protected createToolbarExtensions() {
            super.createToolbarExtensions();
            this.rowSelection = new Serenity.GridRowSelectionMixin(this);
        }

        protected getColumns() {
            var columns = super.getColumns();
            columns.splice(0, 0, Serenity.GridRowSelectionMixin.createSelectColumn(() => this.rowSelection));
            return columns;
        }
`

**Comment by minhhungit** [comment]
you can use GridRadioSelectionMixin

https://github.com/serenity-is/Serenity/wiki/Grid-Radio-Row-Selection-Mixin
